### DISCUSSION TITLE
left bracket bigger than right when vertical bar in brackets using some math fonts

### DISCUSSION THREAD

**Discussion by violet8070** [typst/typst#2161] [General]
### Description

when using some math fonts for example "Linux Libertine",left bracket is bigger than right bracket
code:
```typst
#show math.equation: set text(font: "Linux Libertine")
$
P(|)
$

```
image:
![image](https://github.com/typst/typst/assets/101701076/7965e994-c06b-4ee6-8453-ba38cecfcf12)

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
Linux libertine is not a math font

**Comment by violet8070** [comment]
also encountered this problem when using "Latin Modern Math" @Enivex 
```typst
#show math.equation: set text(font: "Latin Modern Math")
$
P(|)
$
```

**Comment by Enivex** [comment]
> also encountered this problem when using "Latin Modern Math" @Enivex 
> ```typst
> #show math.equation: set text(font: "Latin Modern Math")
> $
> P(|)
> $
> ```
> 

If this is in the web app, it doesn't actually have latin modern math. It's falling back to another font.

**Comment by violet8070** [comment]
I installed latin modern math font and compiled it on my local machine(amd64 archlinux)
typst -V got this:
`typst 0.8.0 (360cc9b9)`
@Enivex

**Comment by Enivex** [comment]
> I installed latin modern math font and compiled it on my local machine(amd64 archlinux)
> typst -V got this:
> `typst 0.8.0 (360cc9b9)`
> @Enivex 

Could you try with fallback: false? I suspect typst can't find the font

**Comment by violet8070** [comment]
Thank you! It shows normal. But It's so strange, `typst fonts` can find "Latin Modern Math"`, why can't typst find the font?

> > I installed latin modern math font and compiled it on my local machine(amd64 archlinux)
> > typst -V got this:
> > `typst 0.8.0 (360cc9b9)`
> > @Enivex
> 
> Could you try with fallback: false? I suspect typst can't find the font
